src/app/modules/global-search/enums/global-search-visible-token.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum GlobalSearchVisibleToken {
+  Failover = 'FAILOVER',
+  Enclosure = 'ENCLOSURE',
+  Vms = 'VMS',
```

Comment:
Maybe we need 'Apps' support here as well?

<img width="682" alt="Screenshot 2024-08-07 at 10 34 07" src="https://github.com/user-attachments/assets/c6e09a40-0502-4a86-8db6-802ad19f30e5">